README.md

```diff
@@ -1,14 +1,49 @@
 # Container Linux Config Transpiler
 
-The config transpiler ("ct" for short) is the utility responsible for transforming a Container Linux instance's configuration from the human-friendly, YAML form into an [Ignition][ignition] configuration. While it is possible to write Ignition configs directly, CoreOS recommends that this tool be used instead. More details about the reasoning and the overall design of Ignition and this project are provided in the [documentation][overview].
+The Config Transpiler ("ct" for short) is the utility responsible for
+transforming a human-friendly Container Linux Config into a JSON file. This
```

Comment:
I was trying to avoid mentioning Ignition too much as it's supposed to be an implementation detail. I can reference it in more places if you want.